wavesrv/pkg/remote/remote.go

```diff
@@ -1886,7 +1896,6 @@ func RunCommand(ctx context.Context, rcOpts RunCommandOpts, runPacket *packet.Ru
 		RunPacket: runPacket,
 	})
 
-	go pushNumRunningCmdsUpdate(&runPacket.CK, 1)
```

Comment:
moved this functionality into AddLineForCmd, which gets called earlier in this function

Any potential issues / regressions here?